willshersystems/ansible-sshd
# Improve test coverage with new test cases and new distros, fixing minor issues on the way

Tests:
 * Ubuntu and Debian CI targets
 * Backup & Restore for each test case to make sure the tests do not depend on each other
 * Verify OS defaults are not changed when role is executed without arguments
 * Verify variable precedence
 * Test role backup functionality
 * RHEL6 compatibility

Fixes:
 * RHEL8 default configuration
 * Debian default configuration
 * Create runtime directory on Debian based systems (for check in case service is not running)
 * Fix vars precedence to work with Debian versions (before, Debian.yml was loaded instead of Debian_9.yml)
 * Clarify match blocks in README
